### DISCUSSION TITLE
Add "--output-dir-template" flag to "helmfile fetch"

### DISCUSSION THREAD

**Discussion by elchenberg** [helmfile/helmfile#453] [Ideas]
It would be great to have more control over the structure of the output directory of the `helmfile fetch --output-dir string` command with a `--output-dir-template` flag.

Currently the default structure seems to be `{{ .OutputDir }}/{{ .Release.Name }}/{{ .Release.Chart }}/{{ .Release.Version }}` but I would like to override it with only `{{ .OutputDir }}`, for example.

(Inspired by the "helmfile template" flag of the same name.)

**Comment by yxxhero** [comment]
@elchenberg PR is welcome.

**Reply by elchenberg** [reply]
Thanks, I'm on it. :slightly_smiling_face:
